src/common/workspace-store.ts

```diff
@@ -138,6 +138,7 @@ export class WorkspaceStore extends BaseStore<WorkspaceStoreModel> {
     super({
       configName: "lens-workspace-store",
     });
+    this.workspaces.get(WorkspaceStore.defaultId).enabled = true;
```

Comment:
You could initialize it when it is created, on line 193

Reply:
Not sure about that.. we cannot pass ` enabled` to `Workspace` constructor because it eats only data that will be persisted.

Reply:
Well you could do the following:

```typescript
  @observable workspaces = observable.map<WorkspaceId, Workspace>({
    [WorkspaceStore.defaultId]: () => {
      const res = new Workspace({
        id: WorkspaceStore.defaultId,
        name: "default"
      });
      res.enabled = true;
      
      return res;
    }()
  });
```

Reply:
Or better yes, initialize this in the constructor instead of at the declaration.

Reply:
I don't get it. `Workspace` has an `enabled` field, and `this.workspaces` is a map of `Workspace`s. Is it because it's observable?

JS/TS complexity starts at 30 for a single statement so it's no wonder that VSCode extension scores our work so high :)

Reply:
I don't understand. The constructor for `Workspace` does not take `enabled` (though the impl would set it). This is to work around that. I don't know why that constructor doesn't take enabled. It seems strange that it doesn't.

Reply:
Oh I see, it copies over only `WorkspaceModel` fields

Reply:
Obstinately yes, though since the code is `Object.assign` it actually copied over everything.

Reply:
more hidden complexity :)

Reply:
Yeah... I don't particularly like `Object.assign` since in the final JS it doesn't do any type checking.